### ISSUE TITLE
Tox not passing environment variables

### ISSUE THREAD

**Issue by 4word** [JBKahn/django-sharding#28]
Tests are failing in Travis because somehow, in the "passenv" section of the tox config, a leading dash is in front of the section name. Not sure how it snuck in, but it shouldn't be there and I believe its causing the tests to not test postgres or mysql specific code, since the default is sqlite if the environment variables are not found. It currently reads:
```
[testenv]
-passenv=
  DATABASE_URL
  SHARD_001_DATABASE_URL
  SHARD_002_DATABASE_URL
  REPLICA_001_DATABASE_URL
  REPLICA_002_DATABASE_URL
  TRAVIS
```

It should read
```
[testenv]
passenv=
  DATABASE_URL
  SHARD_001_DATABASE_URL
  SHARD_002_DATABASE_URL
  REPLICA_001_DATABASE_URL
  REPLICA_002_DATABASE_URL
  TRAVIS
```

**Comment by JBKahn** [comment]
ahhh

**Comment by 4word** [comment]
I think this can be closed now, PR [#29](Ensure travis is testing with database urls. )
